doc/source/doc-style/formatting-tools.rst

```diff
@@ -9,9 +10,9 @@ Most of the tools presented can be configured using :ref:`the
 \`\`pyproject.toml\`\` file`, avoiding dotfiles and thus leading to a much
 cleaner root project directory.
 
-
 Blacken-Docs
```

Comment:
```suggestion
blacken-docs
```
It's odd to have a heading start with a lower case, but this is the case used in their product documentation. You likely wil have to add ``blacken-docs`` to Vale's accept list.